README.md

```diff
@@ -11,4 +14,361 @@ Installation
 Documentation
 -----------------------
 
-Coming soon.
+### Getting Started
+
```

Comment:
up-to-date documentation verified by testing...what is this...I feel so grown up...